sdks/typescript/src/util/workflow-run-ref.ts

```diff
@@ -71,6 +71,15 @@ export default class WorkflowRunRef<T> {
     return this.client.stream(workflowRunId);
   }
 
+  // TODO test this..
```

Comment:
need to javadoc this file and test this method

Reply:
bumping this icymi!